docs/experts/loading-experts.rst

```diff
@@ -0,0 +1,125 @@
+===============
+Loading Experts
+===============
+
+The algorithms in the imitation library are all about learning from some kind of
+expert.
+In many cases it is practical that this expert is a piece of software itself.
```

Comment:
"practical that" sounds a bit off to my ears, possible rephrasing below, there might be others that better capture your intention.
```suggestion
In many cases it is simplest if this expert is a piece of software itself.
```